### ISSUE TITLE
[BUG] I cant lunch any minecraft client i dont know what to do

### ISSUE THREAD

**Issue by Orester190** [PojavLauncherTeam/PojavLauncher#5895]
### Describe the bug

i just put the file on the version file and it said cant download minecraft 

### The log file and images/videos

![Screenshot_2024-08-02-00-15-38-35_bab8561a5f94f0a53956d8cbbeaad774](https://github.com/user-attachments/assets/fcee8b7e-5488-4c6d-99a8-ae9395a6964e)

### Steps To Reproduce

```markdown
idk this part sorry
```

### Expected Behavior

its just said cant download minecraft i expect it to work but it didnt

### Platform

```markdown
- Device model: Oppo reno 5 5g
- CPU architecture:
1 x 2.40 GHz
6 x 1.80 GHz
1 x 2.21 GHz
- Android version: 13
- PojavLauncher version: foxglove-20240727-ae48e8e-v3_open jdk
```

### Anything else?

thats all

**Comment by GaucovyAero** [comment]
click show more, send the log and send the mc version zip or link to the thing here

**Comment by Orester190** [comment]
java.io.IOException: Unable to read Version JSON for version PixelClient 1.8.9
	at net.kdt.pojavlaunch.tasks.MinecraftDownloader.downloadAndProcessMetadata(MinecraftDownloader.java:203)
	at net.kdt.pojavlaunch.tasks.MinecraftDownloader.downloadGame(MinecraftDownloader.java:90)
	at net.kdt.pojavlaunch.tasks.MinecraftDownloader.lambda$start$0$net-kdt-pojavlaunch-tasks-MinecraftDownloader(MinecraftDownloader.java:63)
	at net.kdt.pojavlaunch.tasks.MinecraftDownloader$$ExternalSyntheticLambda0.run(Unknown Source:10)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:644)
	at java.lang.Thread.run(Thread.java:1012)

**Comment by GaucovyAero** [comment]
ok and link?

**Comment by Orester190** [comment]
of the file? wait

**Comment by Orester190** [comment]
https://www.mediafire.com/file/bhtb3v83qhqyawe/Pixel-Client-1.8.9.zip/file?dkey=w4khb97l0p2&r=1095

**Comment by GaucovyAero** [comment]
fixed it on my side, just rename PixelClient 1.8.9 in your .minecraft/versions to just PixelClient
